dbutils_typehint/fs.py

```diff
@@ -2,19 +2,7 @@
 from typing import Optional, Dict, Iterable
 
 
-class FileInfo(abc.ABC):
-    @abc.abstractmethod
-    def name(self) -> str:
-        pass
-
-    @abc.abstractmethod
-    def length(self) -> int:
-        pass
-
-    @abc.abstractmethod
-    def path(self) -> str:
-        pass
-
+class FileInfo(namedtuple('FileInfo', ['path', 'name', 'size'])):
```

Comment:
Thank you, it's a good point that `FileInfo`'s fields are properties.

But there is a remark about your solution. `namedtuple` doesn't inherit `abc.ABC` neither relates with ABCMeta.

I'd rather mark `name`, `length` and `path` as abstract properties:

```python3
    @property
    @abc.abstractmethod
    def name(self) -> int:
        pass
```

Please could you mark the `path`, `name` and `size` as properties?

Reply:
I've updated the PR